docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Docs
+
+This directory contains the stuff for building static html documentations based on [sphinx](https://www.sphinx-doc.org/en/master/).
+
+
+## Build the docs
+Firstly, install the packages:
+
+```sh
+ pip install .[docs]
```

Comment:
Why should user re-install?

Reply:
If directly user try to run the documenation to see the details before installing the QEfficient repo.